design-proposals/migration-target.md

```diff
@@ -0,0 +1,257 @@
+# Overview
+
+We are getting asks from multiple cluster admin that would like - in exceptionel cases - to explicitly specify the "destination" of the VM when doing Live migration.
+While this may be less important in a cloud-native environment,
+we get this ask from many users coming from other virtualization solutions, where this is a common practice.
+The same result can already be achieved today with a few steps, this is only about simplifying it with a single direct API on the single `VirtualMachineInstanceMigration` without the need to alter a VM spec.
+
+As a VM owner I can already constrain my [VM so that it is restricted to run on particular node](https://kubevirt.io/user-guide/compute/node_assignment/) instead of letting the k8s scheduler finding the best node by itself.
+With the same mechanism a k8s user can already [constrain a generic Pod so that it is restricted to run on particular node](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).
+This proposal is aiming to expose the same capability to cluster admins, in a fully declarative way without the need to mess up the spec of the existing VM and revert at the end, when dealing with live migrations.
+
+## Motivation
+In the ideal cloud native design, the scheduler is supposed to be always able to correctly identify
+the best node to run a pod (so the target pod for the VMI after the live-migration) on.
+In the real world, we still see specific use cases where the flexibility do explicitly and directly define the target node for a live migration is a relevant nice-to-have:
+- Experienced admins are used to control where their critical workloads are move to 
+  > I as an admin, notice that a VM with guaranteed resources is having issues (I watched the cpu iowait metric). In order to resolve the performance issue and keep my user happy, I as admin want to move the VM, without interruption, to a node which is currently underutilized - and will make the user's vm perform better.
```

Comment:
We can make this even more convincing if we mention: I'm am a long-time cluster-admin of another, well-established VM management system. Missing this functionality, of one-off placement of a VM on a node, in KubeVirt is one of the reasons I haven't expanded my KubeVirt footprint.

Reply:
I find it hard to be convinced that the project should provide manual solutions to admins instead of an automated, self managed one.

I do find it convincing to provide reasoning like the one @dankenigsberg raised here, just because the admin wants to have control like he had on other projects. It is convincing for me because I have no alternative to give for that need.